src/main/cpp/healthtests/chassistest/ChassisCWTest.cpp

```diff
@@ -0,0 +1,74 @@
+//====================================================================================================================================================
+// Copyright 2025 Lake Orion Robotics FIRST Team 302
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
+// to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE
+// OR OTHER DEALINGS IN THE SOFTWARE.
+//====================================================================================================================================================
+// C++ Includes
+#include <algorithm>
+#include <string>
+#include <iostream>
+
+#include "auton/drivePrimitives/DriveStop.h"
+#include "chassis/HolonomicDrive.h"
+#include "auton/drivePrimitives/DriveStop.h"
+#include "chassis/definitions/ChassisConfigMgr.h"
+#include "ChassisCWTest.h"
+#include "teleopcontrol/TeleopControl.h"
+#include "utils/logging/Logger.h"
+
+ChassisCWTest::ChassisCWTest() : DragonTestCase(std::string("drive"), std::string("CW"))
+{
+    m_timer = new frc::Timer();
+    m_swerve = ChassisConfigMgr::GetInstance()->GetCurrentConfig() != nullptr ? ChassisConfigMgr::GetInstance()->GetCurrentConfig()->GetSwerveChassis() : nullptr;
+    m_drivePathPlanner = new DrivePathPlanner();
+}
+void ChassisCWTest::SetUp()
+{
+
+    m_orignalPose = m_swerve->GetPose();
+
+    m_timer->Reset();
+    m_timer->Start();
+    m_moveInfo.rawX = 1.0;
+    m_moveInfo.rawY = 0.0;
```

Comment:
Not sure you need to set these to non-zero values (definitely rawX is zero and rawOmega would be -1.0; need to set vx, vy and omega